cmd/cluster/create.go

```diff
@@ -73,18 +76,20 @@ func NewCreateCommand() *cobra.Command {
 		Namespace:          "clusters",
 		Name:               "example",
 		ReleaseImage:       releaseImage,
+		BaseDomain:         "example.com",
```

Comment:
Why do we want to set a default for this?  `example.com` doesn't seem like it would ever work outside a local test environment.

Reply:
agree! It was an example but much better force to flag